galaxy/tools/deps/mulled/mulled_build.py

```diff
@@ -17,6 +17,10 @@
 import subprocess
 import sys
 from sys import platform as _platform
+try:
+    from shlex import quote as shlex_quote
+except ImportError:
+    from pipes import quote as shlex_quote
```

Comment:
from six.moves import shlex_quote

Reply:
Right, thanks @nsoranzo, didn't look sideways and thus didn't notice `six`. And how convenient that they also used the generic `shlex_quote`!